### DISCUSSION TITLE
îles with a CMS

### DISCUSSION THREAD

**Discussion by rylanharper** [ElMassimo/iles#124] [General]
Hey there! I really appreciate all the work you've done with this project. I recently migrated over from Gridsome to îles and it works wonderfully when creating a SSG with markdown:)

I am wondering how îles would do when integrated with a CMS? I plan on using the [Sanity-Vue](https://github.com/danielroe/vue-sanity) integration with îles... I am not sure if it will work, but it seems like it should be just fine. Does this project require any specific setup when using a third-party CMS (like Sanity, Prismic, Contentful, etc.) other than what the given docs say? Or does it just function the same way as a regular Vue app? I am somewhat of a newer dev so any info on this would really help!

Thank you!

**Comment by ElMassimo** [comment]
Hi Rylan! @rylanharper 

I'm not familiar with the Sanity CMS, It should be possible to use îles with most CMSs.

During build, îles is effectively a regular Vue app, which runs in SSR mode to generate pages, so as long as the code is compatible with Node.js, it can connect to the internet, fetch third-party data, and use that data to render the pages.

[Here's an example using the Directus API](https://github.com/ElMassimo/iles-directus-example/tree/main/iles).

**Reply by rylanharper** [reply]
Hey @ElMassimo! Okay great thats good to know! It sounds like everything should go just fine setting up îles with a CMS. I also appreciate that Directus repo link. Although I am using Sanity, it has a similar integration pattern as other CMS platforms, and I have messed around with Gridsome + Directus in the past so I will reference this for my project. ~~Perhaps consider adding this example link to the docs under the demos section?~~ Edit: I just found this link on the docs, I should have been more thorough in my reading into this! 😓  

Thanks for the help!:)
